sw/device/tests/coverage_test.c

```diff
@@ -0,0 +1,43 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/test_coverage.h"
+#include "sw/device/lib/testing/test_main.h"
+#include "sw/device/lib/uart.h"
+#include <stdbool.h>
+#include <stdint.h>
+
+static void spin_45(uint8_t state) {
+  static const char *kChars = "|/-\\";
```

Comment:
Optional: name this something like `kSpinnerChars`.

Also, shouldn't this be declared as a `char[]` rather than a `char *`?